src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -46,6 +42,26 @@ function PagesAndResources({ courseId, intl }) {
           >
             <DiscussionsSettings courseId={courseId} />
           </PageRoute>
+          <PageRoute path={`${path}/:appId/settings`}>
+            {
+              ({ match, history }) => {
+                const SettingsComponent = React.lazy(async () => {
+                  try {
+                    // There seems to be an issue with eslint when using template strings here.
+                    return await import('./' + match.params.appId + '/Settings.jsx'); // eslint-disable-line
+                  } catch (error) {
+                    console.trace(error);
+                    return null;
+                  }
+                });
+                return (
+                  <Suspense fallback="...">
```

Comment:
make proper use of the spinner component from paragon. for reference you can check this 
https://paragon-edx.netlify.app/components/spinner/

Reply:
Any update on this one?